plugins/src/android/OptimoveInitProvider.java

```diff
@@ -100,4 +111,32 @@ private String handleNullValues(String credentials) {
         }
         return credentials;
     }
+
+    private DeferredDeepLinkHandlerInterface getDDLHandlerInterface() {
```

Comment:
function name, it returns handler, not an interface

Reply:
I changed the name to getDDLHandler